design-tokens/tokens/layer1/base.json

```diff
@@ -380,21 +380,21 @@
             },
             "type": "typography"
           },
-          "medium": {
+          "semi-bold": {
             "value": {
               "fontFamily": "{echoes.fontFamily.sans}",
-              "fontWeight": "{echoes.fontWeight.medium}",
+              "fontWeight": "{echoes.fontWeight.semi-bold}",
               "lineHeight": "{echoes.lineHeight.20}",
               "fontSize": "{echoes.fontSize.20}",
               "letterSpacing": "{echoes.letterSpacing.default}",
               "paragraphSpacing": "{echoes.paragraphSpacing.medium}"
             },
             "type": "typography"
           },
-          "semi-bold": {
+          "bold": {
             "value": {
               "fontFamily": "{echoes.fontFamily.sans}",
-              "fontWeight": "{echoes.fontWeight.semi-bold}",
+              "fontWeight": "{echoes.fontWeight.medium}",
               "lineHeight": "{echoes.lineHeight.20}",
```

Comment:
Why is `medium` called `bold` for default size, but `medium` for `large`?

Reply:
It should be fixed now too